containers-tests/benchmarks/IntMap.hs

```diff
@@ -123,7 +129,7 @@ main = do
         ]
 
     --------------------------------------------------------
-    !bound = 2^12
+    !bound = 2^14
```

Comment:
Did increasing this demonstrate anything new?

Reply:
I don't so much trust stability of super-short tests (microseconds), which we were getting for the functions in question. If necessary, we can limit the size increase to only some of the benchmarks.

Reply:
> I don't so much trust stability of super-short tests (microseconds)

I think we've discussed this before, but I don't really see why. In case there's noise, I trust the benchmarking library to take care of it and provide a stable result. Anyway, we can adjust the size later if we need to.